env0-resources-generator/terraform_templates/vcs.tftpl

```diff
@@ -0,0 +1,11 @@
+%{ if modules_num > 0 || include_vcs_connection ~}
+data "env0_template" "vcs_connected_template" {
+  name = "Name Of VCS Connected Template"
+}
+
+locals {
+   bitbucket_client_key   = data.env0_template.vcs_connected_template.bitbucket_client_key
+   github_installation_id = data.env0_template.vcs_connected_template.github_installation_id 
```

Comment:
there are more attributes that can be added here, but for example is_azure_devops is not supported by the [provider](https://github.com/env0/terraform-provider-env0/issues/844)

Reply:
It's already added, I think - you should review it BTW :)